controllers/categories.go

```diff
@@ -26,15 +27,19 @@ func (cc *categoryController) CreateCategory(ctx echo.Context) error {
 	var category models.Category
 	if err := ctx.Bind(&category); err != nil {
 		cc.logger.Error(err.Error())
+		log.LogInfo("Invalid request for creating category")
```

Comment:
We don't expect extra abstraction, use `log.Gl.Info` instead.

Reply:
fixed